src/abstract-interpretation/data-frame/mappers/function-mapper.ts

```diff
@@ -67,6 +67,11 @@ const DataFrameFunctionMapper = {
 	'arrange':       { mapper: mapDataFrameIdentity, library: 'dplyr' }
 } as const satisfies Record<string, DataFrameFunctionMapperInfo<never>>;
 
+/**
+ * List of data frame functions that are not explicitly supported but return data frames.
+ */
+const UnsupportedDataFrameFunctions = ['anova', 'lmer', 'Anova', 'Manova', 'AIC', 'BIC'] as const satisfies string[];
```

Comment:
Why not move this into a flowR config so that it can be configured and changed if someone 1) finds new functions or 2) wants to try anyway

Reply:
Additionally, we may want to add package information for the future